### ISSUE TITLE
create pkgbuild repository

### ISSUE THREAD

**Issue by ghost** [archclassroom/archclassroom.github.io#8]
This will be the repository for PKGBUILD classes.

First will need to add the notes from CalimeroTeknik's class. This should include the text on top of the logs files. Then, in a separate commit, the notes meskarune made.

* [ ] Contact CalimeroTeknik and ask for permission to release his notes under the license CC BY-NC-SA 4.0 which we use.
* [ ] Create a new commit with three additional files. The first file is his notes. The second file is the text from the top (and bottom?) of the log file for the first sessions. The third file is the text from the log file for the second session.
* [ ] Create a second commit. Remove the files for CalimeroTeknik's class and put in the notes meskarune drafted.

This way CalimeroTeknik's notes are there in the history and future classes can be built off of meskarune's notes (which were used for Eschwartz's class, iirc).

**Comment by polyzen** [comment]
Blocked by #1.

**Comment by meskarune** [comment]
I don't think we need to create an issue for every class topic. This might possibly be redundant with #1
